drivers/media/platform/bcm2835/bcm2835-camera.c

```diff
@@ -304,6 +304,11 @@ static inline bool is_capturing(struct bm2835_mmal_dev *dev)
 	    component[MMAL_COMPONENT_CAMERA]->output[MMAL_CAMERA_PORT_CAPTURE];
 }
 
+static inline bool capture_port_buffer_empty(struct bm2835_mmal_dev *dev)
```

Comment:
I'm in two minds as to whether this should be here or in mmal-vchiq.c (or .h seeing as it is an inline).

It's looking at a list that is "owned" by mmal-vchiq.c, so really ought to be part of that so that anyone who might update it only has to look in one place.